src/onnxruntime.cc

```diff
@@ -278,6 +283,28 @@ ModelState::LoadModel(
                           value_string, &max_workspace_size_bytes));
                       trt_options.trt_max_workspace_size =
                           static_cast<size_t>(max_workspace_size_bytes);
+                    } else if (param_key == "int8_calibration_table_name") {
+                      RETURN_IF_ERROR(params.MemberAsString(
+                          param_key.c_str(), &int8_calibration_table_name));
+                      trt_options.trt_int8_calibration_table_name = &int8_calibration_table_name;
```

Comment:
```suggestion
                      trt_options.trt_int8_calibration_table_name = int8_calibration_table_name.c_str();
```
It looks like trt_int8_calibration_table_name  is cont char*, and assigning string* to char* doesn't seem right.